tests/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM golang:1.10 AS builder
 WORKDIR /go/src/github.com/nmstate/kubernetes-nmstate/
-RUN go get github.com/onsi/ginkgo/ginkgo
+RUN go get github.com/onsi/ginkgo/ginkgo github.com/ghodss/yaml
```

Comment:
yaml should be acquired through the project vendoring. we have ginkgo here only for its binary/test plugin, not its sources